.dev/format.sh

```diff
@@ -1,4 +1,10 @@
-#! /usr/bin/sh
+#! /usr/bin/env sh
+
+# sample command line usage: $0 clang-format(version >= 6.0) $PCL_SOURCE_DIR
+# $ sh ./.dev/format.sh `which clang-format` ./
```

Comment:
This file is executable and has shebag, so no reason to call with `sh` explicitly?

Reply:
I tried without shebang and the shell threw: "./format.sh: command not found", so I figured, why not, it's just 3 characters for a guarantee. Though in `bash` and `zsh`, it's not an issue

Reply:
Strange.